examples/mpi-mnist/cpu/v1alpha2/mpi_mnist_job_cpu.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "PyTorchJob"
+metadata:
+  name: "pytorch-mpi-mnist-cpu"
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/akado2009/pytorch-mpi-mnist-cpu
+    Worker:
+      replicas: 3
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/akado2009/pytorch-mpi-mnist-cpu
```

Comment:
this has to be changed when the image is uploaded to official kubeflow registry (like other examples)